api/routes/units.js

```diff
@@ -0,0 +1,91 @@
+const router = require('express').Router()
+const Units = require('../models/units')
+const Company = require('../models/company')
+
+
+//Get all routes and queries. 
+router.get('/units', async (req, res, next) => {
+    const status = 200
+
+     if(typeof req.query.kind !== 'undefined'){
+         await Units.find().select('kind').then(response => {
+            res.json({ status, response })
+         })
+
+    } else if (typeof req.query.floor !== 'undefined') {
+        await Units.find().select('floor').then(response => {
+            res.json({ status, response })
+         })
+
+    } else if (typeof req.query.occupied !== 'undefined') {
```

Comment:
It doesn't look like you're actually using the true/false value from `req.query.occupied`.